### ISSUE TITLE
Keyword Matching in Alt Text fails when keyword has a period at the start AND another word in front of it in the alt text

### ISSUE THREAD

**Issue by Pcosta88** [Yoast/YoastSEO.js#1124]
## What did you expect to happen?
Yoast to find the Focus keyword in the Meta Description

## What happened instead?
It did not. The warning of "The images on this page do not have alt attributes containing the focus keyword." appeared

## How can we reproduce this behavior?
**1. Make keyword be ".Net Developer"**

**2. Make Alt text be "Download .Net Developer Resume"**

![screen_shot_2017-03-30_at_3_18_35_pm](https://cloud.githubusercontent.com/assets/8238074/24522096/74eb8100-155c-11e7-844f-4a58e4daf41a.png)

**3. See Warning: The images on this page do not have alt attributes containing the focus keyword.**

![screen shot 2017-03-30 at 3 18 52 pm](https://cloud.githubusercontent.com/assets/8238074/24522077/662acab8-155c-11e7-8f1d-4364cb0a7628.png)

## Technical info

* Platform: WordPress 
* Platform version: 4.7.3
* Yoast SEO version: 4.5

**Comment by Pcosta88** [comment]
Related: https://github.com/Yoast/YoastSEO.js/issues/1123

**Comment by Pcosta88** [comment]
Please inform the customer of conversation # 188014 when this conversation has been closed.

**Comment by Pcosta88** [comment]
Please inform the customer of conversation # 196093 when this conversation has been closed

**Comment by Pcosta88** [comment]
Closed. Issue does not happen in v14.8.1 of Yoast
